src/change/sdp.rs

```diff
@@ -335,6 +354,14 @@ impl<'a> SdpApi<'a> {
 
         id
     }
+
+    /// Perform an ICE restart.
+    pub fn ice_restart(&mut self) {
```

Comment:
Return the new `IceCreds` here.

Reply:
My thinking was that there was no way to get a hold of the local credentials previously so it felt consistent to keep them hidden. If we return them here shouldn't there be a way to get them via `Rtc`?